quikey/quikey.py

```diff
@@ -92,6 +95,35 @@ def ls(ctx, show_all):
     output = AsciiTable(table)
     click.echo(output.table)
 
+@cli.command()
+@click.option('--location', '-l', default=BaseDirectory.xdg_config_home+"/autokey/data/", show_default=True, help='Location of top level directory to import from autokey')
+@click.pass_context
+def keyimport(ctx,location):
+    tags = ['autokey-imports']
+    db = ctx.obj['database']
+    importfiles = PhraseFind(location)
+    for key in importfiles:
+        contents = None
+        if importfiles[key] is not None:
+            if db.get(name) is not None:
```

Comment:
Needs to be `db.get(key)`

Reply:
Good catch. I also realize that the following line needed to have "key" instead of "name" so I resolved that as well.